src/CMakeLists.txt

```diff
@@ -255,18 +298,21 @@ add_executable(avrdude
     whereami.h
     "${EXTRA_WINDOWS_SOURCES}"
     )
-
-target_link_libraries(avrdude PUBLIC libavrdude)
+if (sharedlib IN_LIST BUILT_LIBS)
+    target_link_libraries(avrdude PUBLIC sharedlib)
+else()
+    target_link_libraries(avrdude PUBLIC staticlib)
+endif()
```

Comment:
Should we retain the old behaviour of linking `avrdude` executable with the static library?

Reply:
Ping @dl8dtl

Reply:
Ping @mariusgreuel